src/HonzaBotner.Services.Contract/IRoleBindingsService.cs

```diff
@@ -5,7 +5,7 @@ namespace HonzaBotner.Services.Contract
 {
     public interface IRoleBindingsService
     {
-        Task<IList<ulong>> FindMappingAsync(ulong channelId, ulong messageId, string emojiName);
+        Task<IList<ulong>> FindMappingAsync(ulong channelId, ulong messageId, string? emojiName = null);
```

Comment:
Why does one have `= null` and the other not?

![image](https://user-images.githubusercontent.com/5287596/108569103-98c7d680-730b-11eb-8ca1-3204cf7080a8.png)

Reply:
Hmm, because I didn't update the other one? It doesn't really affect the code execution, but I will add it.

Reply:
Fixed.